addons/hr_holidays/models/hr_employee.py

```diff
@@ -182,16 +182,17 @@ def _search_absent_employee(self, operator, value):
         ])
         return [('id', 'in', holidays.mapped('employee_id').ids)]
 
-    @api.model
-    def create(self, values):
-        if 'parent_id' in values:
-            manager = self.env['hr.employee'].browse(values['parent_id']).user_id
-            values['leave_manager_id'] = values.get('leave_manager_id', manager.id)
-        if values.get('leave_manager_id', False):
-            approver_group = self.env.ref('hr_holidays.group_hr_holidays_responsible', raise_if_not_found=False)
-            if approver_group:
-                approver_group.sudo().write({'users': [(4, values['leave_manager_id'])]})
-        return super(HrEmployeeBase, self).create(values)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if 'parent_id' in vals:
+                manager = self.env['hr.employee'].browse(vals['parent_id']).user_id
+                vals['leave_manager_id'] = vals.get('leave_manager_id', manager.id)
+            if vals.get('leave_manager_id', False):
+                approver_group = self.env.ref('hr_holidays.group_hr_holidays_responsible', raise_if_not_found=False)
+                if approver_group:
+                    approver_group.sudo().write({'users': [(4, vals['leave_manager_id'])]})
+        return super().create(vals_list)
```

Comment:
```suggestion
    def create(self, vals_list):
        approver_group = self.env.ref('hr_holidays.group_hr_holidays_responsible', raise_if_not_found=False)
        for vals in vals_list:
            if 'parent_id' in vals:
                manager = self.env['hr.employee'].browse(vals['parent_id']).user_id
                vals['leave_manager_id'] = vals.get('leave_manager_id', manager.id)
            if approver_group and vals.get('leave_manager_id', False):
                approver_group.sudo().write({'users': [(4, vals['leave_manager_id'])]})
        return super().create(vals_list)
```

Reply:
Might aswell batch the write 🙂 
```suggestion
    def create(self, vals_list):
        approver_group = self.env.ref('hr_holidays.group_hr_holidays_responsible', raise_if_not_found=False)
        group_updates = []
        for vals in vals_list:
            if 'parent_id' in vals:
                manager = self.env['hr.employee'].browse(vals['parent_id']).user_id
                vals['leave_manager_id'] = vals.get('leave_manager_id', manager.id)
            if approver_group and vals.get('leave_manager_id'):
                group_updates.append((4, vals['leave_manager_id']))
        if group_updates:
            approver_group.sudo().write({'users': group_updates})
        return super().create(vals_list)
```